pandas/tools/tile.py

```diff
@@ -258,12 +283,16 @@ def _format_levels(bins, prec, right=True,
     else:
         levels = ['[%s, %s)' % (fmt(a), fmt(b))
                   for a, b in zip(bins, bins[1:])]
-
     return levels
 
 
-def _format_label(x, precision=3):
+def _format_label(x, precision=3, dtype=None):
     fmt_str = '%%.%dg' % precision
+
+    if dtype == np.datetime64:
```

Comment:
@sinhrks, i can do that, that means that i would not have to pass the dtype as an argument, right? (I can check the dtype using is_datetime64_dtype call). Though it would mean that i make this call twice, once early on, when qcut or cut is called and later in _format_label, i mean if the call is not expensive, it shouldnt make a lot of difference